File: backend/models/dinosaur.py
from sqlalchemy import ForeignKey, Table, Column
from sqlalchemy.sql.sqltypes import Integer, String, Float
from config.db import meta, engine

dinosaurs = Table("dinosaurs", meta, 
    Column("id", Integer, primary_key=True), 
    Column("name", String(255)),  
    Column("species", Integer, ForeignKey("species.id")),
    Column("age", Integer), 
    Column("weight", Float), 
    Column("gender", Integer, ForeignKey("genders.id")))

meta.create_all(engine)

File: frontend/lib/models/dinosaur.dart
import 'package:frontend/models/gender.dart';
import 'package:frontend/models/species.dart';

class Dinosaur {
  int id;
  String name;
  Species species;
  int age;
  double weight;
  Gender gender;

  Dinosaur(
      this.id, this.name, this.species, this.age, this.weight, this.gender);
}
